README.md

```diff
@@ -42,8 +45,11 @@ You are welcome to review code / UX of addons and provide that feedback directly
 
 - Submissions and automated checks should be easy to find and get the status of.
 - Make it possible to automate many steps in the process.
-- Once submitted an addon version should be immutable. No changing the addon on an external server, it must be updated in the store via the submission process.
-- Allow addon authors to easily revoke a version if it is buggy / no longer supported. It should no longer be presented in the store, halting installation by new users.
+- Submitted add-on release data should be immutable.
+  No changing the addon on an external server and using the same URL.
```

Comment:
I attempted to word this more passively:
```suggestion
 A `.nvda-addon` file accessed by its unique download URL cannot be changed.
```

Reply:
I think this can be reworded, but I want to be clear what the implications are. Since addon authors are in control of the data at the URL, they may attempt to change the data. But the SHA will no longer match and NVDA will refuse to install it.